components/tidb_qe/src/executor/run.rs

```diff
@@ -0,0 +1,316 @@
+// Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::sync::Arc;
+
+use kvproto::coprocessor::KeyRange;
+use protobuf::Message;
+use tipb::executor::{self, ExecType, ExecutorExecutionSummary};
+use tipb::select::{Chunk, DAGRequest, SelectResponse, StreamResponse};
+
```

Comment:
Remove this extra blank line.

Reply:
The blank line is intended, for separating external dependencies and project dependencies.